rdagent/scenarios/kaggle/developer/runner.py

```diff
@@ -71,6 +71,35 @@ def develop(self, exp: KGModelExperiment) -> KGModelExperiment:
 
         return exp
 
+class MLEModelRunner(KGCachedRunner[KGModelExperiment]):
+    @cache_with_pickle(KGCachedRunner.get_cache_key, KGCachedRunner.assign_cached_result)
```

Comment:
I think we can use the previous runner; it's just that if MLE is doing the benchmark, we need to submit the submission.csv file to the Grade server.